tech.json

```diff
@@ -1650,6 +1650,14 @@
             "It is often beneficial to be in a downward pose when hitting the ceiling, in order to shrink Samus' hitbox to delay contact."
           ]
         },
+        {
+          "name": "canMomentumConservingUnmorph",
```

Comment:
How about calling this `canMomentumConservingMorph` so that the cases we have for morphing can be included, like crab hole and draygon.  And unmorphing being another way of using that tech.

Reply:
Sounds good, updated it to use that.